docs/developer/app-store/apps/segment.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Twilio Segment
+sidebar_label: Twilio Segment
+sidebar_position: 6
+---
+
+import { AppMetadata } from "/components/AppMetadata/AppMetadata.jsx";
+
+<AppMetadata
+  minSaleorVersion="3.20"
+  githubUrl="https://github.com/saleor/apps/tree/main/apps/segment"
+/>
+
+This application integrates with [Twilio Segment](https://segment.com/). It tracks events occurring in Saleor and sends them to Twilio Segment.
+
+## Features
+
+- Send information about order-related events to Segment.
+- Events send to Segment have identity set to user ID or email.
+
+## Assumptions & limitations
+
+-  Currently, only order-related events are supported.  
+-  Order webhooks are asynchronous, meaning events may not be sent to Segment immediately.  
+-  The app uses the [`track` API](https://segment.com/docs/connections/sources/catalog/libraries/server/node/#track). 
+-  The app attempts to retrieve user information using the user ID or email. If that fails, it will send an anonymous ID.  
+-  The app has its webhooks initially disabled; configuring the app will activate the webhooks.  
+
+## Permissions
+
+- `MANAGE_ORDERS`
+
+## Application flow
+
+The application subscribes to the following webhooks:
+
+- `ORDER_CREATED`
+- `ORDER_CANCELLED`
+- `ORDER_FULLY_PAID`
+- `ORDER_REFUNDED`
+- `ORDER_UPDATED`
+
+For each webhook, the application will send the following information to Segment
+
+- Order `id` in base64 format
+- User information:
+    - User `id` in base64 format
```

Comment:
question: Is this exactly `Order.id` from Saleor or do we do some additional `base64` encoding?

I'm wondering if we should even mention it's base64, since it can change at any time, because Saleor didn't guarantee that it will use this specific format and we **definitely** don't want to recommend users to decode that base64 because they will start depending on it.

Reply:
exactly `Order.id` 😄 

Good question - maybe just `Order.id` is enough

Reply:
Done in [`572718b` (#1435)](https://github.com/saleor/saleor-docs/pull/1435/commits/572718b6585d8628c1cf645f93e210f8a8e6b945)